tests/sqllogictests/suites/udf_native/03_0001_udf_py.test

```diff
@@ -0,0 +1,16 @@
+## enable it when compiled with ee feature
+## statement ok
```

Comment:
We don't need these tests?

Reply:
we don't enable python udf by default, so we comment these for now.